libs/s25main/Game.cpp

```diff
@@ -94,8 +96,9 @@ void Game::RunGF()
 
     if(world_.HasLua())
         world_.GetLua().EventGameFrame(em_->GetCurrentGF());
-    // Update statistic every 750 GFs (30 seconds on 'fast')
-    if(em_->GetCurrentGF() % 750 == 0)
+    // Update statistic every 30 seconds
+    constexpr unsigned GFsIn30s = 30 * 1000 / SPEED_GF_LENGTHS[referenceSpeed];
```

Comment:
@Flow86 I'm not sure if we should change that. Correct would be every 37,5s (on normal)
Or wait... Did we have our statics time (i.e. the "last 15min") always based on fast? Looks like it... O.o
Ok, so this comment just for reference that this (correctly) changes the reference time for the statistic

Reply:
Yes, it was based on fast before